dts/bindings/input/st,stmpe811.yaml

```diff
@@ -0,0 +1,127 @@
+# Copyright (c) 2023 Antmicro <www.antmicro.com>
+# SPDX-License-Identifier: Apache-2.0
+
+description: STMPE811 I2C touchscreen controller
+
+compatible: "st,stmpe811"
+
+include: i2c-device.yaml
+
+properties:
+  int-gpios:
+    type: phandle-array
+    description: |
+      Interrupt GPIO. Used by the controller to signal touch data is
+      available. Active low.
+
+  screen-width:
+    type: int
+    default: 0
+    description: |
+      Screen width for scaling the reported coordinates.
+      Default: raw touchscreen resolution.
+
+  screen-height:
+    type: int
+    default: 0
+    description: |
+      Screen height for scaling the reported coordinates.
+      Default: raw touchscreen resolution.
+
+  raw-x-min:
+    type: int
+    default: 0
+    description: |
+      Signed raw X axis start for scaling the reported coordinates.
+      No effect if screen size is not set.
+      Default: no offset.
+
+  raw-y-min:
+    type: int
+    default: 0
+    description: |
+      Signed raw Y axis start for scaling the reported coordinates.
+      No effect if screen size is not set.
+      Default: no offset.
+
+  raw-x-max:
+    type: int
+    default: 4096
+    description: |
+      Raw X axis end for scaling the reported coordinates.
+      No effect if screen size is not set.
+      Default: raw touchscreen resolution.
+
+  raw-y-max:
+    type: int
+    default: 4096
+    description: |
+      Raw Y axis end for scaling the reported coordinates.
+      No effect if screen size is not set.
+      Default: raw touchscreen resolution.
+
+  panel-driver-settling-time-us:
+    type: int
+    enum:
+      - 10
+      - 100
+      - 500
+      - 1000
+      - 5000
+      - 10000
+      - 50000
+      - 100000
+    default: 1000
+    description: |
+      Panel driver settling time (microseconds). For large panels (> 6"), a capacitor of 10 nF
+      is recommended at the touchscreen terminals for noise filtering.
+      In this case, settling time of 1 ms or more is recommended.
```

Comment:
what if the default condition is not met? sounds like this property needs to be required

Reply:
We changed it, now `panel-driver-settling-time-us` is required, with no default value.